IPs/RSKIP170.md

```diff
@@ -0,0 +1,87 @@
+# Peg-in to any address
+
+|RSKIP          |170           |
+| :------------ |:-------------|
+|**Title**      |Peg-in to any address |
+|**Created**    |01-SEP-20 |
+|**Author**     |MI |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes a way for users to transfer BTC to RSK, indicating the RSK address in which they want their RBTCs transferred to and a BTC refund address in case the peg-in can not be completed for some reason.
+
+## Motivation
+
+Allowing users to lock funds to any address in RSK, EOA or contract.
+
+## Specification
+
+This RSKIP proposes an extension to the current peg-in workflow. By including an output with an OP_RETURN op code and certain payload in the transaction sent to the federation to perform the peg-in, the user can determine the RSK address where the RBTCs will be transferred to and a BTC refund address. 
+
+The BTC transaction sent to the federation must contain 
+ one output with value 0 and OP_RETURN op code followed by the following data:
+- `52534b54` as prefix indicating the output is for RSK. This values stands for `RSKT` (for RSK  transfer) in ascii and hex encoded. (4 bytes)
+- Protocol version in hexa (2 bytes). Currently **01**
```

Comment:
Why in hexa and not a single binary byte?

Reply:
If there is a strong reason, it should be in the rationale part

Reply:
To allow having multiple versions of the protocol, as we add new functionalities (contract execution, UTXOs consolidation, whatever may come next). We can reduce this field to just 1 byte, it still allows us to have 16 different protocol versions. More than enough I think.